api/v1alpha1/datatable_types.go

```diff
@@ -59,6 +59,9 @@ type DataTableStatus struct {
 	// the data of mount points have been mounted
 	Schemas []Schema `json:"mounts,omitempty"`
 
+	// the alluxio master IP
+	CacheMasterIP string `json:"cacheMasterIP,omitempty"`
```

Comment:
Is it a good way to bind data table to alluxio? I think some other runtimes also have the similar function.

Reply:
I am not  sure whether juicefs or other runtime supples the jar package for hive or other query engine. If they do not supply the packer, we need to use fuse container to access data.  Maybe it is a good way to implement it in the future.